packages/steps/src/cacheUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { bunyan } from '@expo/logger';
+
+export interface Cache {
+  disabled: boolean;
+  clear: boolean;
```

Comment:
What do these do?

Reply:
Unresolving discussion. It remains unclear what does `disabled` or `clear` mean in this context.